packages/cloudflare/src/cli/config.ts

```diff
@@ -11,6 +11,9 @@ const UserConfig = {
 };
 
 export type Config = {
+  // Timestamp for when the build was started
+  timestamp: number;
```

Comment:
nit: can you rename to `buildTimestamp`

Reply:
wouldn't it `buildStartTimestamp` be clearer?

making the variable self descriptive and avoiding the need for the extra comment

Reply:
Oops sorry @dario-piotrowicz I merged before seeing your comment.
I feel like `buildTimestamp` is good enough but feel free to update